services/exposurelog/values-summit.yaml

```diff
@@ -0,0 +1,21 @@
+exposurelog:
+
+  nfs_path_1: /readonly/comcam
+  nfs_server_1: comcam-arctl01.cp.lsst.org
+  butler_uri_1: /volume_1  # a guess
+
+  ingress:
+    enabled: true
+    annotations: {}
+    #   kubernetes.io/ingress.class: nginx
+    #   kubernetes.io/tls-acme: "true"
+    hosts:
+      - host: lsst-nts-k8s.ncsa.illinois.edu
+        paths:
+          - /exposurelog
+    tls: []
+    #  - secretName: chart-example-tls
+    #    hosts:
+    #      - chart-example.local
+
+vault_path: secret/k8s_operator/lsst-nts-k8s.ncsa.illinois.edu/postgres
```

Comment:
and the vault_path should be secret/k8s_operator/summit-lsp.lsst.codes/postgres and I confirm that the exposurelog_password is there.

Reply:
DItto